tests/unit/s2n_safety_macros_test.c

```diff
@@ -15,7 +15,6 @@
  */
 
 #include "s2n_test.h"
-
 #include "utils/s2n_safety.h"
 
 /**
```

Comment:
The comment right below here says DO NOT DIRECTLY MODIFY THIS FILE.
It's the automatically generated tests for our automatically generated utils/s2n_safety_macros.h